### PR TITLE
[new] md-tabs improvements : md-tooltip

### PR DESCRIPTION
**Pull request by jtouzy**
Improvement in md-tabs : Can display a tooltip (md-tooltip) on the tab header.
+ Doc update

### CODE REVIEW

File: src/components/mdTabs/mdTab.vue [hunk 1]
```diff
@@ -14,7 +14,9 @@
       mdLabel: [String, Number],
       mdIcon: String,
       mdActive: Boolean,
-      mdDisabled: Boolean
+      mdDisabled: Boolean,
+      mdTooltip: String,
+      mdTooltipDirection: String
```
Comment (marcosmoura) [inline src/components/mdTabs/mdTab.vue]: The direction default should be set here to not make the getTabData do the dirty check everytime.

File: src/components/mdTabs/mdTab.vue [hunk 2]
```diff
@@ -54,6 +62,8 @@
           icon: this.mdIcon,
           active: this.mdActive,
           disabled: this.mdDisabled,
+          tooltip: this.mdTooltip,
+          tooltipDirection: this.mdTooltipDirection || 'bottom',
```
Comment (marcosmoura) [inline src/components/mdTabs/mdTab.vue]: The same here.

**Comment by marcosmoura** [comment]
The branch has a conflict. Could you fix this first?

**Comment by jtouzy** [comment]
@marcosmoura I changed the default value initialization with the last commit.
But for the conflit, i'm not very familiar with the git-pull requests so I don't really know how to do this. I think the conflit is only with the latest updates (the last pull request I made), but maybe i'm wrong?

**Comment by marcosmoura** [comment]
Can you make sure to sync your fork [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [here](https://help.github.com/articles/syncing-a-fork/).

**Comment by jtouzy** [comment]
It seems to be ok right now :-)

**Pull request merged**